src/widgets/ticket-detail/constants/seat-map.mock.ts

```diff
@@ -0,0 +1,12 @@
+export type SeatMapMock = {
+  id: number;
+  seatImageUrl: string;
+};
+
+const SEAT_MAP_MOCK_MAP: Record<number, SeatMapMock> = {
+  1: { id: 1, seatImageUrl: "/img_seatview_118.webp" },
+  2: { id: 2, seatImageUrl: "/img_seatview_410.webp" },
+  3: { id: 3, seatImageUrl: "/img_seatview_507.webp" },
+};
```

Comment:
개별 id가 있으니

```suggestion
const SEAT_MAP_MOCK_MAP: Record<number, SeatMapMock> = [
  { id: 1, seatImageUrl: "/img_seatview_118.webp" },
  { id: 2, seatImageUrl: "/img_seatview_410.webp" },
  { id: 3, seatImageUrl: "/img_seatview_507.webp" },
];
```
이렇게 사용하는게 나을것 같아요